src/main/data_structure/stack.py

```
@@ -0,0 +1,41 @@
+class Stack:
+
+    # Constructor to initiate stack
+    def __init__(self, stack=None):
+        if stack is None:
+            stack = []
+        self.stack = stack
+
+    # this returns the top item of stack
+    def top(self):
+        if Stack.empty(self) is False:
+            return self.stack[-1]
+        else:
+            print("An empty stack")
```

raise an exception instead of printing An empty Queue
https://www.w3schools.com/python/gloss_python_raise.asp#:~:text=Raise%20an%20exception,exception%2C%20use%20the%20raise%20keyword